dashboard/migrations/0001_initial.py

```diff
@@ -133,7 +122,8 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='User',
             fields=[
-                ('id', models.CharField(max_length=255, primary_key=True, serialize=False, verbose_name='User Id')),
+                ('id', models.AutoField(primary_key=True, serialize=False, verbose_name='Table Id')),
+                ('user_id', models.CharField(blank=False, max_length=255, null=False, verbose_name='User Id')),
                 ('name', models.CharField(max_length=255, verbose_name='Name')),
```

Comment:
In `views.py` the call to `get_user_assignment_submission` function need to change to grab the `user_id` instead of `id`
```
sql = "select assignment_id, score, graded_date from submission where " \
          "user_id=(select **user_id** from user where sis_name = %(current_user)s and course_id = %(course_id)s ) and course_id = %(course_id)s"
```

Reply:
Good catch, #506 should refactor this table and use this id directly, but until that happens I think it will have to link up as you describe still.